File: llm_outputs/codellama13b/GuruSamples/csharp/extract-class_refactored.cs
class Soldier 
{
  public int Health { get; set; }
  public int Damage { get; set; }
  public int WeaponStatus { get; set; }
  
  public int GetDamage() 
  {
    return Damage;
  }
  public void Attack() 
  {
    Health -= GetDamage();
  }
}

File: results/llm_outputs/codellama7b/GuruSamples/php/extract-class_refactored.php
<?php
class Soldier {
  public $health; // int
  public $damage; // int
  public $weaponStatus; // int

  public function getDamage() {
    return $this->damage;
  }

  public function attack() {
    $this->health -= $this->getDamage();
  }
}
